lib/scroll-through-time.js

```diff
@@ -35,30 +43,42 @@ function getListener (direction) {
 export default {
 
   config: {
-    autoToggle: {
-      title: 'Auto Toggle',
-      description: 'Toggle on start.',
+    active: {
+      title: 'Scroll History Active',
+      description: 'Enable on Atom startup.',
       type: 'boolean',
       default: true
     },
     direction: {
       title: 'Scroll direction',
+      description: 'Set the scroll direction action. Must be up/down on devices without touchpad or sidescroll.',
       type: 'string',
-      default: 'left',
+      default: 'down',
       enum: [
         {value: 'left', description: 'Scroll left to undo, right to redo'},
         {value: 'right', description: 'Scroll right to undo, left to redo'},
         {value: 'up', description: 'Scroll up to undo, down to redo'},
         {value: 'down', description: 'Scroll down to undo, up to redo'},
       ],
+    },
+    keyboardState: {
+      title: 'Keyboard + Mouse Support',
+      description: 'Enable support for a keyboard modifier key + scroll.',
+      type: 'boolean',
```

Comment:
Could we change this do be an enum directly? Maybe `none` (default), `cmd`, `control`, `option`?